cfme/common/provider.py

```diff
@@ -79,10 +81,10 @@ def form(self):
                         version.LOWEST: Input('bearer_password'),
                         '5.6': Input('default_password')
                     }),
+                    ('token_verify_secret', Input('bearer_verify')),
                     ('token_verify_secret', {
-                        version.LOWEST: Input('bearer_verify'),
-                        '5.6': Input('default_verify')
-                    }),
+                        version.LOWEST: Input('bearer_password'),
```

Comment:
@jkrocil is right, the list of form fields is processed sequentially, so having two lines with the same "key" is wrong.